spec/models/job_run_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe JobRun, type: :model do
+  context "validation" do
+    let(:user) do
+      User.new(sunet_id: "Jdoe")
+    end
+
+    let(:bc) do
+      BundleContext.new(project_name: "SmokeTest",
+                        content_structure: 1,
+                        bundle_dir: "spec/test_data/bundle_input_g",
+                        staging_style_symlink: false,
+                        content_metadata_creation: 1,
+                        user: user)
+    end
+
+    let(:discovery_report_run) do
+      JobRun.new(output_location: "Report available at /path/to/report",
```

Comment:
i'd be inclined to just have this be the path, and to have any human readable part be done at the last possible moment at the presentation layer.  makes it easier to do mechanical things with the field, e.g. validate that the directory is there without additional string parsing.

(path only is of course allowed by the code as-is, but would be nice to have the tests exemplify that usage too)

Reply:
see #253